src/EditorConfig.Core/EditorConfigParser.cs

```diff
@@ -91,13 +91,13 @@ public FileConfiguration Parse(string fileName, IEnumerable<EditorConfigFile> ed
 			var sections =
 				from configFile in editorConfigFiles
 				from section in configFile.Sections
-				where IsMatch(section.Glob, fullPath, configFile.Directory)
+				where IsMatch(section.Glob, fullPath)
 				select section;
 
 			return new FileConfiguration(ParseVersion, file, sections.ToList());
 		}
 
-		private bool IsMatch(string glob, string fileName, string directory)
+		private bool IsMatch(string glob, string fileName)
```

Comment:
"directory" parameter wasn't used here. So I removed it just to be sure that EditorConfig without a directory can be used as well. I.e. matching section glob directly to the file name.